skbio/io/format/genbank.py

```diff
@@ -429,7 +431,6 @@ def _parse_single_genbank(chunks):
         elif header == 'FEATURES':
             # merge the two dictionaries together. Note this is only
             # available in Python-3.5
-
             feature_metadata = merge_dicts(parsed, feature_metadata)
         else:
             metadata[header] = parsed
```

Comment:
Can you clarify this? The `metadata` object is just a dictionary, it doesn't correspond to the features/intervals in the `interval_metadata` object.

Also what if `locus_name` is not present?  The `id` will still be empty ...